projects/angular/src/wizard/_wizard.clarity.scss

```diff
@@ -267,6 +315,21 @@
         variables.$clr-use-custom-properties
       );
 
+      &:not(.disabled):hover .clr-wizard-stepnav-link {
+        @include mixins.css-var(
+          color,
+          string.unquote('clr-wizard-stepnav-text--active'),
```

Comment:
I know and understand why have to be like that, but we have to rename this property after token migration.

Reply:
Yep, totally agree :)